src/reynir_correct/errtokenizer.py

```diff
@@ -1872,6 +1872,10 @@ def add_ritmyndir_error(token: CorrectToken) -> CorrectToken:
             return token
         _, m = db.lookup_g(corrected, at_sentence_start=at_sentence_start)
         text, details, refs = get_details(code, token.txt, corrected, m[0].stofn)
+        if not details:
+            # Code not found in collection
+            print("Code not found: {}".format(code))
```

Comment:
Delete print statement, and/or return an error condition in some other way?

Reply:
Added AssertionError that prints the message.

Reply:
Got it. Deleted the message. This has been a problem when we retrieve new Ritmyndir data, as the (text, details, refs) data is not in that but formed from the metadata from the website, which would then need to be updated. This could be addressed by updating the metadata everytime we retrieve new Ritmyndir data, which happens at the most monthly. We could add a tool that compares the categories in the Ritmyndir data to what is found in the metadata, I can look into that.
Anyway, this should no longer be a problem here, if we don't find it in the metadata we just do nothing.

Reply:
-- this tool was added in the newest commit and proved a great help!